src/item.cpp

```diff
@@ -1100,1290 +1100,1355 @@ faction_id item::get_old_owner() const
-                if( bipod && parts->test( iteminfo_parts::GUN_RECOIL_BIPOD ) ) {
-                    info.emplace_back( "GUN", "bipod_recoil", _( " (with bipod <num>)" ),
-                                       iteminfo::lower_is_better | iteminfo::no_name,
-                                       aprox->gun_recoil( g->u, true ) );
-                }
+    if( has_ammo ) {
+        // ammo_damage, sum_of_damage, and ammo_mult not shown so don't need to translate.
+        if( mod->ammo_data()->ammo->prop_damage ) {
+            if( parts->test( iteminfo_parts::GUN_DAMAGE_AMMOPROP ) ) {
+                info.push_back( iteminfo( "GUN", "ammo_mult", "*",
+                                          iteminfo::no_newline | iteminfo::no_name,
+                                          *mod->ammo_data()->ammo->prop_damage ) );
             }
         } else {
-            if( mod->gun_recoil( g->u ) ) {
-                if( parts->test( iteminfo_parts::GUN_RECOIL ) ) {
-                    info.emplace_back( "GUN", _( "Effective recoil: " ), "",
-                                       iteminfo::no_newline | iteminfo::lower_is_better,
-                                       mod->gun_recoil( g->u ) );
-                }
-                if( bipod && parts->test( iteminfo_parts::GUN_RECOIL_BIPOD ) ) {
-                    info.emplace_back( "GUN", "bipod_recoil", _( " (with bipod <num>)" ),
-                                       iteminfo::lower_is_better | iteminfo::no_name,
-                                       mod->gun_recoil( g->u, true ) );
-                }
+            if( parts->test( iteminfo_parts::GUN_DAMAGE_LOADEDAMMO ) ) {
+                damage_instance ammo_dam = has_ammo ? curammo->ammo->damage : damage_instance();
+                info.push_back( iteminfo( "GUN", "ammo_damage", "",
+                                          iteminfo::no_newline | iteminfo::no_name |
+                                          iteminfo::show_plus, ammo_dam.total_damage() ) );
             }
         }
-        info.back().bNewLine = true;
-
-        std::map<gun_mode_id, gun_mode> fire_modes = mod->gun_all_modes();
-        if( std::any_of( fire_modes.begin(), fire_modes.end(),
-        []( const std::pair<gun_mode_id, gun_mode> &e ) {
-        return e.second.qty > 1 && !e.second.melee();
-        } ) ) {
-            info.emplace_back( "GUN", _( "Recommended strength (burst): " ), "",
-                               iteminfo::lower_is_better,
-                               ceil( mod->type->weight / 333.0_gram ) );
+        if( parts->test( iteminfo_parts::GUN_DAMAGE_TOTAL ) ) {
+            info.push_back( iteminfo( "GUN", "sum_of_damage", _( " = <num>" ),
+                                      iteminfo::no_newline | iteminfo::no_name,
+                                      mod->gun_damage( true ).total_damage() ) );
         }
+    }
 
-        if( parts->test( iteminfo_parts::GUN_RELOAD_TIME ) ) {
-            info.emplace_back( "GUN", _( "Reload
```

Comment:
is_gun() was only needed here because `type->mod` is not currently being instantiated for items that are gun mods without being of type `GUNMOD`

Reply:
Thanks, fixed!